data/Clojure-replDemoData.xml

```diff
@@ -19,5 +19,5 @@ under the License.
 -->
 
 <entity-engine-xml>
-
+<SystemProperty replPort="99999999999" />
```

Comment:
Pune un port ok - nu aia. 
Putem folosi 7888.

Lipsește listen address - adresa la care asculta - implicit localhost.

Putem specifica tipul de date pentru proprietate? port ar trebui sa fie int (Integer) dacă se poate.

Reply:
Am schimbat portul, am adaugat adresa la care sa asculte : localhost.
In entitymodel.xml putem specifica tipul de date pentru proprietati, am pus portul integer.